modules/testkit/src/test/scala/com/caesars/tracing/TracingTestUtils.scala

```diff
@@ -0,0 +1,187 @@
+package com.caesars.tracing
+
+import cats.effect.kernel.Ref
+import io.janstenpickle.trace4cats.`export`.RefSpanCompleter
+import io.janstenpickle.trace4cats.inject.EntryPoint
+import io.janstenpickle.trace4cats.kernel.{SpanCompleter, SpanSampler}
+import io.janstenpickle.trace4cats.model.{CompletedSpan, SpanKind, TraceHeaders, TraceProcess}
+import sttp.client3
+import sttp.client3.impl.zio.RIOMonadAsyncError
+import sttp.client3.{Identity, RequestT, SttpBackend, UriContext, basicRequest, Response as SttpResponse}
+import sttp.client3.testing.SttpBackendStub
+import sttp.model.RequestMetadata
+import zhttp.http.*
+import zhttp.service.Server
+import zio.interop.catz.*
+import zio.interop.catz.implicits.rts
+import zio.{Has, Task, UIO, ULayer, URLayer, ZIO, ZLayer}
+
+import scala.collection.immutable.Queue
+
+object TracingTestUtils {
+
+  val refSpanCompleter: ULayer[Has[RefSpanCompleter[Task]]] =
+    RefSpanCompleter[Task]("my-service").toLayer.orDie
+
+  def entryPointRef: ZLayer[Has[RefSpanCompleter[Task]], Nothing, Has[EntryPoint[Task]]] =
```

Comment:
Non-blocking question: is `entrypointRef` the only "public" field of `TracingTestUtils`? If so, is it worthwhile, in the name of minimizing the surface area of the API, to put the rest of the `val/def`'s in `object Internal { ... }`?

I was going to suggest `private`, but I believe you prefer using the `object Internal` approach, Alex. I also think your approach is more appropriate for open-source, i.e. where users shouldn't necessarily be hindered from accessing fields (which would happen with `private`).